tests/system/marathon_common_tests.py

```diff
@@ -1078,6 +1078,22 @@ def http_output_check():
     http_output_check()
 
 
+@shakedown.dcos_1_10
+def test_network_name_with_underscore_is_deployed():
+    """Tests that application containing network name with underscore is accepted
+        by Marathon service.  We do not assert that the app is deployed because that
+        would require to create that network prior to running tests."""
```

Comment:
Wouldn't that also crash the agents when you try to deploy an app with a non-existing cni network?

Reply:
this is docker, not UCR so no CNI and thus hopefully no crash...

but thanks for reminding me, I need to push that stacktrace to core-oncall...